app/backend/src/couchers/servicers/api.py

```diff
@@ -464,13 +472,17 @@ def ListFriendRequests(self, request, context):
         with session_scope() as session:
             sent_requests = (
                 session.query(FriendRelationship)
+                .join(User, User.id == FriendRelationship.to_user_id)
+                .filter(User.is_visible)
```

Comment:
I wonder if it's possible to do something like `.filter(FriendRelatoinship.to_user.is_visible)`? Might be some SA magic that we could use? Though maybe joins are nice and explicit?

Reply:
Tried it, got this error: https://stackoverflow.com/questions/16589208/attributeerror-while-querying-neither-instrumentedattribute-object-nor-compa

Suggested "workaround" is using joins

Reply:
Thanks for trying. Joins are probably more explicit anyway!